test/truffle/ecosystem/launchers.sh

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 
-source test/truffle/common.sh.inc
+# get the absolute path of the executable and resolve symlinks
+SELF_PATH=$(cd "$(dirname "$0")" && pwd -P)/$(basename "$0")
+while [ -h "$SELF_PATH" ]; do
+    # 1) cd to directory of the symlink
+    # 2) cd to the directory of where the symlink points
+    # 3) get the pwd
+    # 4) append the basename
+    DIR=$(dirname "$SELF_PATH")
+    SYM=$(readlink "$SELF_PATH")
+    SELF_PATH=$(cd "$DIR" && cd "$(dirname "$SYM")" && pwd)/$(basename "$SYM")
+done
```

Comment:
Isn't this test always called with CWD=truffleruby repo?

Reply:
I've run into this not being fixed, so I made it more robust.

Reply:
So what is a case where it's run with CWD != the truffleruby repo?

Reply:
I've run it manually from some directory. Any script like that is better when it works from anywhere.

Reply:
OK, although not being able to assume CWD often makes the script more complex.
Doesn't `check_launchers bin/ true` assume it's run in the Ruby home?
This is worth a comment, since the Ruby home is not always the same as the truffleruby repo (e.g. testing in GraalVM).

Reply:
So actually `jt test ecosystem` won't work in GraalVM then because it use `sh` which `chdir` to the TruffleRuby repo used for testing, which is not GraalVM's Ruby home.

Reply:
Maybe the script should `cd `jt ruby -e 'puts Truffle::Boot.ruby_home'` then.

Reply:
working on it